argparse.go

```diff
@@ -229,6 +229,24 @@ func (o *Command) List(short string, long string, opts *Options) *[]string {
 	return &result
 }
 
+// Positional creates an argument which is a catch-all for any remaining strings left over after parsing.
+// Returns a pointer to the list of strings.
+func (o *Command) Positional(long string, opts *Options) *PositionalResult {
```

Comment:
No need for very first argument here, only options is enough. Because below we can do:

``` go
a := &arg{
  ...
  lname: "positional",
  ...
}
```

Reply:
The purpose of that first argument is to provide a user-visible name that is later used in the help text.  I've improved the documentation and example code to make this more clear.